update by 14th of May

Drsheppard01 opened the pull request.

blueOkiris commented on pkgs.json:
```diff
@@ -8,7 +8,7 @@
         "name": "acreom",
         "version": "1.5.8",
```
Forgot to version bump here. Should be 1.6.4

blueOkiris commented on pkgs.json:
```diff
@@ -56,59 +61,74 @@
     }, {
         "name": "audacity",
-        "version": "3.2.5",
+        "version": "3.3.0",
```
Should say 3.3.2 I believe

blueOkiris reviewed (changes requested):
Needs just a couple fixes I think

Drsheppard01 commented:
> Hey, FYI I just updated aip-man and added the feature to install from local repos.
> You might find this useful when doing these updates as you can test changes and things now before ever posting.
> To install from a different repo simply do `aipman -r <custom repo> install <package>`
> You can do this locally to test stuff by prepending the file path to the repo.json with `file://`

I had previously thought of this when using the NX Software Center in Nitrux (Linux distro that emphasizes AppImage). This is a very cool feature

I'm also currently working on an AppImage database something like an alternative to appimage.github.io or appimagehub.com consisting of aip-man pkg list
